### PR TITLE
Updated UI for facility cards and avatar component

### PR DESCRIPTION
**Pull request by shivankacker**
## Proposed Changes

image

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
image

In the updated UI, when the bed capacity exceeds the limit, the text turns into a red badge, which I feel may not look good. I suggest that when the facility is over bed capacity, the text should be made bold and the danger color applied, which might resemble the style shown in the screenshot below.

image

@shivankacker can you ask around the team for a opinion and let me know, which will the updated UI

**Comment by shivankacker** [comment]
@nihal467 thought of this too, let me ask the team

**Comment by nihal467** [comment]
@shivankacker once the PR is ready for test, change the label

**Comment by shivankacker** [comment]
@nihal467 as discussed on today's weekly review, we can stick to the current UI

**Comment by nihal467** [comment]
@shivankacker can you clear the merge conflict

**Comment by nihal467** [comment]
LGTM

File: src/Components/Common/Avatar.tsx [hunk 1]
```diff
@@ -44,43 +45,54 @@ const initials = (name: string): string => {
...
 const Avatar: React.FC<AvatarProps> = ({
   colors: propColors,
   name,
+  imageUrl,
   className,
-  square = false, // Default to false for backwards compatibility
 }) => {
-  const [bgColor, fgColor] = propColors || toColor(name);
+  const [bgColor] = propColors || toColor(name);
+  const [w, setW] = useState(0);
```
Comment (khavinshankar) [inline src/Components/Common/Avatar.tsx]: lets rename this variable to width and setWidth

File: src/Components/Common/Avatar.tsx [hunk 2]
```diff
@@ -44,43 +45,54 @@ const initials = (name: string): string => {
...
+  const avatarRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    const updateW = () => {
```
Comment (khavinshankar) [inline src/Components/Common/Avatar.tsx]: this too to, updateWidth

**Review by khavinshankar — COMMENTED** [review]
@shivankacker LGTM, just some minor variable renaming

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**